src/pretix/presale/forms/checkout.py

```diff
@@ -31,6 +38,35 @@ def __init__(self, *args, **kwargs):
                 help_text=_('Please enter the same email address again to make sure you typed it correctly.'),
             )
 
+        if self.event.settings.order_phone_asked:
+            babel_locale = 'en'
+            # Babel, and therefore django-phonenumberfield, do not support our custom locales such das de_Informal
```

Comment:
```suggestion
            # Babel, and therefore django-phonenumberfield, do not support our custom locales such as de_Informal
```
Nur ein Mini-Typo. Sorry, will auf keinen Fall pedantisch sein.

Reply:
Keine Sorge :)